drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -344,6 +346,12 @@ def report_item_fields_from_enrollment(report_items:, enrollment:, report:) # ru
       report_item
     end
 
+    private def enrollment_cocs(report)
+      @enrollment_cocs ||= self.class.enrollment_scope(report).
+        pluck(e_t[:id], ec_t[:CoCCode]).
+        to_h
+    end
```

Comment:
Does this need to be smarter? I'm worried that it uses a report parameter, but that isn't considered when it is memoized?

Reply:
report is the report object calling the calculation, it's passed down so we can get the report scope.  Since this is memoized for the entire enrollment scope and indexed on enrollment.id, it should be fine.